### DISCUSSION TITLE
Replace box/spout with openspout/openspout

### DISCUSSION THREAD

**Discussion by ankurk91** [spatie/simple-excel#88] [Ideas]
Hi,

The `box/spout` project is no longer maintained.
https://github.com/box/spout

I would suggest to use this one instead.

https://github.com/openspout/openspout

**Comment by freekmurze** [comment]
Feel free to send a PR for this! If the tests still pass, I'd certainly consider merging it in.

**Reply by neklein** [reply]
@ankurk91 are you planning on creating a pull request on this?

**Reply by ankurk91** [reply]
https://github.com/spatie/simple-excel/pull/90

Already merged

**Reply by neklein** [reply]
Ah, my bad! I missed the release!
